packages/headless/src/features/commerce/parameters/parameters-serializer.ssr.ts

```diff
@@ -0,0 +1,335 @@
+import {isNullOrUndefined} from '@coveo/bueno';
```

Comment:
Adapted from https://github.com/coveo/ui-kit/blob/master/packages/headless/src/features/search-parameters/search-parameter-serializer.ssr.ts

Where possible and simple, I'm reusing util functions.

The most different part was dealing with sort criteria.